server/migrations/20210803222419-add-2v2-matchmaking-type.js

```diff
@@ -0,0 +1,85 @@
+exports.up = async function (db) {
```

Comment:
Do we need to support anything prior to Postgres 9.1? I don't think we really do, that's pretty old at this point (and way older than what we use on the server). Thus you can do this a lot more simply (and not need to update every single table that uses it):

https://stackoverflow.com/questions/1771543/adding-a-new-value-to-an-existing-enum-type/7834949#7834949

Reply:
Guess you can't really do the same to remove it in the reverse migration though, so maybe it's not really that much of a benefit

Reply:
Ah, annoying :( Guess we're stuck doing this forever then